sysutils/pfSense-pkg-node_exporter/Makefile

```diff
@@ -0,0 +1,39 @@
+# $FreeBSD$
+
+PORTNAME=	pfSense-pkg-node_exporter
+PORTVERSION=	0.18.1
+PORTEPOCH=	1
+CATEGORIES=	sysutils
+MASTER_SITES=	# empty
+DISTFILES=	# empty
+EXTRACT_ONLY=	# empty
+
+MAINTAINER=	coreteam@pfsense.org
+COMMENT=	pfSense package node_exporter
+
+LICENSE=	APACHE20
+
+RUN_DEPENDS=	${LOCALBASE}/bin/node_exporter:sysutils/node_exporter
```

Comment:
This dependency format it deprecated, instead of pointing to a file just use the package. It will be like this:
```RUN_DEPENDS=	node_exporter>0:sysutils/node_exporter```